src/main/java/webserver/RequestHandler.java

```diff
@@ -21,35 +21,35 @@ public void run() {
 
         try (InputStream in = connection.getInputStream(); OutputStream out = connection.getOutputStream()) {
             //입력으로 들어온 내용을 매개변수로 받아 request 객체 생성
-            Request request = Request.getRequest(in);
-            DataOutputStream dos = new DataOutputStream(out);
+            Request request = new Request(in);
             Response response;
+            DataOutputStream dos = new DataOutputStream(out);
             RequestController requestController = RequestController.getRequestController();
 
-            //request가 바로 view와 연결될 수 있을 때
+            //정적 데이터 조회
             if (requestController.isMappingView(request)) {
```

Comment:
```suggestion
            if (requestController.isRequestStatic(request)) {
```
isMappingView가 뭔가 했더니 정적응답을 확인하는 메서드군요.